test/test_util.py

```diff
@@ -0,0 +1,10 @@
+import json
+
+
+class TestUtils:
+
+    @staticmethod
+    def read_json_file(file_path: str):
+        file = open('./data/{0}'.format(file_path))
```

Comment:
as @dzleidig mentioned in his comment, this should be relative to the root directory

Reply:
Fixed